### PR TITLE
 Fixes #25738 - use valid host names in fixtures

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
This is just adding one additional commit to https://github.com/Katello/katello/pull/7864

File: db/seeds.d/101-locations.rb [hunk 1]
```diff
@@ -4,17 +4,15 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 
-if Location.exists? || ENV['SEED_LOCATION'].present?
```
Comment (johnpmitsch) [inline db/seeds.d/101-locations.rb]: whats the reasoning behind removing this? (just curious)

Comment (jlsherrill) [reply db/seeds.d/101-locations.rb]: See: https://github.com/Katello/katello/pull/7864

change in foreman.

**Comment by tbrisker** [comment]
```
Failure:
Katello::Api::Rhsm::CandlepinDynflowProxyControllerIntegrationTest#test_0001_params are not parsed in the controller [/var/lib/workspace/workspace/katello-pr-test/test/controllers/api/rhsm/candlepin_dynflow_proxy_controller_integration_test.rb:17]:
not all expectations were satisfied
unsatisfied expectations:
- expected never, invoked once: JSON.parse(any_parameters)
satisfied expectations:
- allowed any number of times, not yet invoked: Katello::Ping.ping(any_parameters)
- allowed any number of times, not yet invoked: Cert::Certs.ssl_client_key(any_parameters)
- allowed any number of times, not yet invoked: Cert::Certs.ssl_client_cert(any_parameters)
- allowed any number of times, not yet invoked: Cert::Certs.ca_cert(any_parameters)
- expected exactly once, invoked once: #<AnyInstance:Katello::Api::Rhsm::CandlepinDynflowProxyController>.async_task(any_parameters)
- expected exactly once, invoked once: Katello::Resources::Candlepin::Consumer.get(any_parameters)


bin/rails test /var/lib/workspace/workspace/katello-pr-test/test/controllers/api/rhsm/candlepin_dynflow_proxy_controller_integration_test.rb:16

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
So this is now failing on the 'db:seed' portion of the test pipeline.  Since an org is now created always, the katello seed is trying to create it in candlepin which won't exist on a unit test run.  We might could move the seed test to an actual test file rather than a pipeline phase, and then stub out the calls to candlepin (i'll give this a try).  
@ehelms curious your thoughts on this?

**Comment by jlsherrill** [comment]
Going ahead and merging this since it fixes the unit tests and should unblock the release pipeline.

**Pull request merged**